packages/modeldb-idb/src/api.ts

```diff
@@ -59,6 +60,59 @@ export class ModelAPI {
 		await this.getStore(txn).delete(key)
 	}
 
+	async count(txn: IDBPTransaction<any, any, IDBTransactionMode>, where?: WhereCondition): Promise<number> {
+		const store = this.getStore(txn)
+
+		if (!where) {
```

Comment:
if someone passes in an empty object for the where condition, we should still use the simple `store.count()`. this is likely to happen since we tend to use default init params (`where = {}`) a lot throughout the codebase